src/hooks/useSSO.ts

```diff
@@ -0,0 +1,24 @@
+import { useCallback } from "react";
+import firebase from "firebase/app";
+
+import { SSO_PROVIDER_ID } from "secrets";
+
+export const useSSO = () => {
+  const hasSSOProvider = SSO_PROVIDER_ID !== undefined;
```

Comment:
We should pass in `SSO_PROVIDER_ID` as a prop here, and read it off the `venue` config itself rather than a const like this.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1546/commits/cd6ed2a394579a1a482db8a751630924672ad528